components/mintmaker/development/renovate-config.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
```

Comment:
This update can be applied to "components/mintmaker/base/renovate-config.yaml" directly.